docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -35,26 +102,28 @@ WINDOW 8
 HORIZON 4;  
 ```
 
-You can check the status of the predictor:
+## Checking the Status of a Predictor
+
+You can check the status of the predictor by executing the folowing SQL command:
 
 ```
 SELECT * FROM mindsdb.predictors where name='home_sales_model';
 ```
 
-## Generating some forecasts
+## Making Predictions
 
-Once the predictor has been successfully trained, you can query it to get forecasts for a given period of time. Usually, you'll want to know what happens right after the latest training data point that was fed, for which we have a special bit of syntax, the "LATEST" key word:
+Once the predictor has been successfully trained, you can query it to get forecasts for a given period of time. Usually, you'll want to know what happens right after the latest training data point that was fed. We have a special bit of syntax for that: the `LATEST` keyword.
 
 ```
```

Comment:
Missing sql flavour on code block.

Reply:
Could you please give me some details on that?